src/test/java/core/basesyntax/strategy/impl/AddOperationServiceImplTest.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AddOperationServiceImplTest {
+
+    @Before
+    public void setUp() throws Exception {
+        Storage.clear();
+        Storage.add(new Fruit("apple"), 20);
+        Storage.add(new Fruit("banana"), 30);
+        Storage.add(new Fruit("orange"), 33);
+    }
+
+    @Test
+    public void addOperationServiceImpl_addOperationTest_Ok() {
+        AddOperationServiceImpl addOperationService = new AddOperationServiceImpl();
```

Comment:
Create for this object field in class and initialize it in @Before
```suggestion
        AddOperationServiceImpl addOperationService = new AddOperationServiceImpl();
```